test/chainsaw/testmetrics/dynatrace-provider/00-install.yaml

```
@@ -0,0 +1,23 @@
+apiVersion: metrics.keptn.sh/v1beta1
+kind: KeptnMetric
+metadata:
+  name: podtatometric
+  namespace: keptn-system
+spec:
+  provider:
+    name: "dynatrace"
+  query: "query"
+  fetchIntervalSeconds: 5
+
+---
+apiVersion: metrics.keptn.sh/v1beta1
+kind: KeptnMetricsProvider
+metadata:
+  name: dynatrace
+  namespace: keptn-system
+spec:
+  secretKeyRef:
+    key: DT_TOKEN
+    name: dynatrace
+  type: dynatrace
+  targetServer: "http://localhost:8080"  # string
```

this should point to our mock server service so something like this should do the trick
```suggestion
  targetServer:  (join('.', ['http://mockserver', $namespace, 'svc.cluster.local:1080']))
```